search/search_cache.go

```diff
@@ -0,0 +1,37 @@
+package search
+
+import(
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+  "github.com/jellydator/ttlcache/v2"
+)
+
+type SearchCache struct {
+  Cache ttlcache.SimpleCache
+}
+
+func (c *SearchCache) Get(query Query, sortBy string, from int, size int) (*QueryResult, error) {
+  key := query.ToFullSimpleString(sortBy, from, size)
```

Comment:
Is `query.ToFullSimpleString` stable enough ?
Remember that all logically equivalent queries should be mapped to exactly the same cache key